content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -471,7 +466,14 @@ Reference_ (under _Pipeline Syntax_) after Jenkins loads and uses that library
 as part of a successful Pipeline run.
 ====
 
-=== Defining steps
+.Avoid preserving state in global variables
+[WARNING]
+====
+Avoid defining global variables with methods that interact or preserve state.
+Use a static class or instantiate a local variable of a class instead.
```

Comment:
This is so vague as to be incomprehensible to me at least.

Reply:
@jglick  You objected to providing an example, but you also feel the text is to vague.  Which would you prefer?